app/services/user_authentication/user_login_logout.py

```diff
@@ -20,23 +20,23 @@
 from app.services.output_manager.message_handler import SrvOutPutHandler
 
 
-def exchange_api_key(api_key: str) -> Union[str, None]:
+def exchange_api_key(api_key: str) -> Union[Tuple[str, str], None]:
     """Exchange API Key with JWT token using Keycloak."""
 
     url = f'{AppConfig.Connections.url_keycloak_realm}/api-key/{api_key}'
     try:
         response = requests.get(url, timeout=5)
         response.raise_for_status()
     except RequestException:
-        return None
+        return None, None
 
-    return response.json()['access_token']
+    return response.json()['access_token'], response.json()['refresh_token']
```

Comment:
I would suggest to avoid parsing json response twice:
```suggestion
    response = response.json()
    return response['access_token'], response['refresh_token']
```